src/Microsoft.VisualStudio.Editors/MyApplication/MyApplicationCodeGenerator.vb

```diff
@@ -31,7 +31,22 @@ Namespace Microsoft.VisualStudio.Editors.MyApplication
         Private _codeDomProvider As CodeDomProvider
         Private _serviceProvider As ServiceProvider
 
+        ' Constants for properties to be generated.
         Private Const MyNamespaceName As String = "My"
+        Private Const _mainFormFieldName As String = "MainForm"
```

Comment:
In the WinForms repo, we're defining Consts starting with a capital letter. 
Should this be done here as well?

Reply:
Yes! Adding in next commit.